core/src/main/java/hudson/security/HudsonAuthenticationEntryPoint.java

```diff
@@ -102,6 +102,7 @@ public void commence(ServletRequest request, ServletResponse response, Authentic
             }
             out.printf(
                 "<html><head>" +
+                "<meta http-equiv='Content-Type' content='text/html;charset=utf-8'>" +
                 "<meta http-equiv='refresh' content='1;url=%1$s'/>" +
```

Comment:
If someone came with same error,you can see here
https://github.com/containous/traefik/issues/1060